.trivyignore

```diff
@@ -0,0 +1,7 @@
+# Ignore wordpress know vulnerabilities for version 5.5.9 (last mayor version 5)
```

Comment:
"Ignore WordPress known vulnerabilities for version 5.5.9 (last major version 5)"?

Reply:
Yes, there're some PHP libs that have some CVEs. Version 5.5.9 is the latest version 5.X.X for now.

Reply:
ah, sorry :D

Reply:
Fixed